components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1742,9 +1742,9 @@ public void deleteGroup(String groupId) throws NotFoundException, CharonExceptio
         } else if (startIndex != 1 && count >= 0) {
             throw new NotImplementedException("Pagination is not supported");
         } else if (rootNode != null) {
-            return filterGroups(rootNode, requiredAttributes);
+            return filterGroups(rootNode, startIndex, count, sortBy, sortOrder, domainName, requiredAttributes);
```

Comment:
Do we use this method having the count argument as `int`?

Reply:
This would enable implementing pagination and sorting in future versions.  Changing the method signature would not disturb the original flow.